_data/sites.json

```diff
@@ -888,6 +888,17 @@
         ]
     },
 
+    {
+        "name": "Alelo",
+        "url": "https://www.alelo.com.br/meucartao/duvidas-frequentes",
+        "difficulty": "easy",
+        "notes": "On app, go to 'Settings' -> 'Delete Account'. Follow the requested.",
+        "notes_pt_br": "NO app, vá até 'Opções' -> 'Excluir conta'. Siga o solicitado.",
+        "domains": [
+            "alelo.com"
```

Comment:
Shouldn't this be `alelo.com.br`?

`alelo.com` and `alelo.com.br` seem to be run by different companies